README.md

```diff
@@ -60,18 +60,18 @@ If you don't know what Deus Ex Randomizer is, then here's our old trailer which
     * Takes away 1 random augmentation, and 1 of your weapons
     * Takes away half your skill points and 5 random skill levels, so the player has to choose which ones to level up first
 * New game modes! [(Wiki page)](https://github.com/Die4Ever/deus-ex-randomizer/wiki/Game-Modes)
-    * Entrance Randomization - changes what level each teleporter takes you to, but keeps it within the same mission.
-    * Horde Mode - fight for your life to see how long you can survive in the Paris Cathedral.
+    * Entrance Randomization - changes what level each teleporter takes you to, but keeps it within the same mission
+    * Horde Mode - fight for your life to see how long you can survive in the Paris Cathedral
     * Zero Rando - great for first-time Deus Ex players to benefit from the bug fixes, QoL improvements, and balance changes we've made, without any randomization.
     * Randomizer Lite - randomizes some things without interfering with the immersion and mood of the game. Great for players who haven't played Deus Ex in a long time, or if you're intimidated by the full Randomizer.
     * Serious Sam mode - same as the normal game but with 10x as many enemies. The player has increased health and takes reduced damage to compensate.
-    * WaltonWare mode - a game mode focused on quickly doing bingos and New Game+.
-    * Speedrun mode - speedrun with fewer resets while still being able to enjoy higher difficulties. And a built in splits viewers!
+    * WaltonWare mode - a game mode where you start in a random mission and win by completing one bingo. As New Game+ keeps making it harder, see how fast you can complete them or how many you can complete!
```

Comment:
I like all of your changes here except I still think we should include some mention of how quick and non-committal WW can be